src/boxes/movie.rs

```diff
@@ -124,17 +125,21 @@ impl Mp4Box for Movie {
             fields.push(mvhd.as_ref());
         }
 
-        for track in self.tracks.iter() {
-            fields.push(track.as_ref());
-        }
-
         if let Some(udta) = self.udta.as_ref() {
             fields.push(udta.as_ref());
         }
 
+        for track in self.tracks.iter() {
```

Comment:
Again, why the order of the code is changed? Any particular reason?

Reply:
See: 
1. https://github.com/musec/fmvparse/pull/3/files/40f4dfc53032079fad8adfa10b9a0881a93e22bb#r872698293
2. https://github.com/musec/fmvparse/pull/3/files/40f4dfc53032079fad8adfa10b9a0881a93e22bb#r872700454
3. https://github.com/musec/fmvparse/pull/3/files/40f4dfc53032079fad8adfa10b9a0881a93e22bb#r872695330